netcdf4/src/main/java/ucar/nc2/jni/netcdf/Nc4prototypes.java

```diff
@@ -104,9 +104,12 @@ Use this in mode flags for both nc_create() and nc_open(). */
  *  by nc_inq_format. 
  */
   int NC_FORMAT_CLASSIC = (1);
-  int NC_FORMAT_64BIT = (2);
+  int NC_FORMAT_64BIT_OFFSET = (2);
+  int NC_FORMAT_64BIT = (NC_FORMAT_64BIT_OFFSET); // Alias
   int NC_FORMAT_NETCDF4 = (3);
   int NC_FORMAT_NETCDF4_CLASSIC = (4);
+  int NC_FORMAT_64BIT_DATA = (5);
+  int NC_FORMAT_CDF5 = (NC_FORMAT_64BIT_DATA); // alias
```

Comment:
We may not need these changes either. Or, at the least, these variables should be assigned the values from `NCheader`. Or possibly vice versa. Basically, I'm complaining about duplication of constant values.